### DISCUSSION TITLE
Idea for the future: enable matching by strata

### DISCUSSION THREAD

**Discussion by m-qin** [NSAPH-Software/CausalGPS#203] [Ideas]
Right now, in my ADRD project with Dan and Naeem, I have written code to do this by breaking GPS estimation and matching into two separate steps, where the second step is done within strata but uses the (unstratified/globally) estimated GPS from the first step.

**Comment by vicitm44444** [comment]
![image](https://user-images.githubusercontent.com/79038343/233794688-b0d3fbd6-6945-479e-90fb-b7e8ac275531.png)
